pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
+---
+
+The AsyncAPI document is a YAML or JSON file that describes your implementation of the AsyncAPI specification. Providing a standardized way to document and describe asynchronous and event-driven systems, the AsyncAPI document generates documentation/code and defines the different components of your event-driven application, such as the events, channels, and message formats.
+
+Primarily, it provides a method to produce comprehensive, consistent, and standardized AsyncAPI documentation. Additionally, it serves as a contract for communication between consumers and producers in an event-driven system. It specifies what should be included in the payload when a service produces a message and provides information to the consumer about the properties in the message. 
```

Comment:
> Primarily, it provides a method to produce comprehensive, consistent, and standardized AsyncAPI documentation. 

I think this is already stated in the previous sentence:

> Providing a standardized way to document and describe asynchronous and event-driven systems

Reply:
agreed, thanks for calling that out